src/assets/styles/tailwind.css

```diff
@@ -466,10 +466,6 @@ input:-ms-input-placeholder, textarea:-ms-input-placeholder {
   color: #a0aec0;
 }
 
```

Comment:
these css rules were removed automatically somehow - I'm guessing you can ignore them?